packages/manager/modules/nasha/src/components/modal-form/modal-form.template.html

```diff
@@ -0,0 +1,31 @@
+<div
+    class="nasha-components-modal-form modal d-block"
+    tabindex="-1"
+    data-ng-mouseup=":: $ctrl.close()"
+>
+    <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+        <div class="modal-content">
+            <form
+                name="$ctrl.form"
+                role="form"
+                data-ng-submit=":: $ctrl.doSubmit()"
```

Comment:
better to have `on` as prefix for events handler.
```suggestion
                data-ng-submit=":: $ctrl.onSubmit()"
```